resources/r/model.R

```diff
@@ -87,19 +87,30 @@ join_prevalence_data_to_shapefile <- function(regional_prevalence_with_ranks, re
 #Order by rank
 regional_prevalence_with_ranks$Parent.Name <- factor(regional_prevalence_with_ranks$Parent.Name, 
                                                      levels = regional_prevalence_with_ranks$Parent.Name[order(regional_prevalence_with_ranks$prevalence)])
```

Comment:
Does the order by rank bit need to be included within the function for the bar chart?  Or maybe this is a separate issue for a different branch.